src/styles/categories.css

```diff
@@ -0,0 +1,36 @@
+.categories__list {
+  list-style: none;
+  display: flex;
+  gap: 70px;
+  padding: 32px;
+  border-bottom: 1px solid var(--lightgray);
+}
+
+.categories__item {
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  background-color: transparent;
+  border: none;
+  color: var(--lightgray);
+  font-size: 1.125rem;
+  cursor: pointer;
+}
+
+/* Tentei dar um query selector para colocar as classes 'categories__item, categories__icon e categories__text' no evento click, porém só funcionava para o primeiro elemento
+e como no momento não sou grande coisa em JS e o pedido foi para que não o usasse, deixei essas estilizações no hover mesmo que não seja o pedido no figma */
+
+.categories__item:hover{
+  color: var(--white);
+  cursor: pointer;
+}
+
+.categories__icon:hover{
+  filter: brightness(0) saturate(100%) invert(100%) sepia(0%) saturate(1941%) hue-rotate(238deg) brightness(117%) contrast(100%);
```

Comment:
Bem interessante a forma que você encontrou para resolver isso... Eu estava pensando em algo mais voltado a esse artigo https://css-tricks.com/change-color-of-svg-on-hover/. Depois me manda a referência que você encontrou pra resolver com filter. 

Só ficou algo um pouco mais verboso e de repente no longo prazo fica mais difícil de dar manutenção.

Aqui, também acho que rola de trabalhar com seletor composto, porque o ícone só muda de cor quando de ato a gente faz o hover nele, o comportamento padrão seria fazer o cover no item como um todo e tanto ícone quanto text mudarem de cor.

Reply:
A ferramenta foi essa aqui, peguei em algum comentário no stackoverflow, é só colocar a cor que deseja para o svg que é sucesso https://codepen.io/sosuke/pen/Pjoqqp